.github/workflows/test.yml

```diff
@@ -121,6 +121,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
+      - uses: mymindstorm/setup-emsdk@v14
+        if: matrix.target == 'wasm32-unknown-emscripten'
```

Comment:
These lines are not needed I think?

Reply:
They are: https://github.com/chronotope/chrono/actions/runs/8430535599/job/23086540259?pr=1541